src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery firstRandomBall = new Lottery();
+        Lottery secondRandomBall = new Lottery();
+        Lottery thirdRandomBall = new Lottery();
+
+        System.out.println(firstRandomBall.getRandomBall());
+        System.out.println(secondRandomBall.getRandomBall());
+        System.out.println(thirdRandomBall.getRandomBall());
```

Comment:
We dont need to create 3 instances of lottery class, we can create only one and call `getRandomBall()` method for several times on it (use loop for this purpose)